static/styles/left_sidebar.css

```diff
@@ -164,6 +164,10 @@ li.show-more-topics {
     .inactive_stream {
         opacity: 0.5;
     }
+
+    i {
+        opacity: 0.7;
+    }
```

Comment:
We don't want to reduce the opacity of every icon in the streams' section of the left sidebar. Make sure this only applies to the ellipsis icon.

Reply:
Alright, thanks for your feedback . I will work on it.  Also one quick question.. 
The ellipsis icon is just the three dots icon ?